polaris-react/build/esm/styles.css

```diff
@@ -611,6 +611,29 @@ html.p-theme-light-high-contrast-experimental {
 }
 :root {
   --polaris-version-number: '12.21.0';
+  --pg-navigation-width: 15rem;
+  --pg-dangerous-magic-space-4: 1rem;
+  --pg-dangerous-magic-space-5: 1.25rem;
+  --pg-dangerous-magic-space-8: 2rem;
+  --pg-layout-width-primary-min: 30rem;
+  --pg-layout-width-primary-max: 41.375rem;
+  --pg-layout-width-secondary-min: 15rem;
+  --pg-layout-width-secondary-max: 20rem;
+  --pg-layout-width-one-half-width-base: 28.125rem;
+  --pg-layout-width-one-third-width-base: 15rem;
+  --pg-layout-width-nav-base: 15rem;
+  --pg-layout-width-page-content-partially-condensed: 28.125rem;
+  --pg-layout-width-inner-spacing-base: 1rem;
+  --pg-layout-width-outer-spacing-min: 1.25rem;
+  --pg-layout-width-outer-spacing-max: 2rem;
+  --pg-layout-relative-size: 2;
+  --pg-dismiss-icon-size: 2rem;
+  --pg-top-bar-height: 3.5rem;
+  --pg-mobile-nav-width: calc(100vw - 2rem - 2rem);
+  --pg-control-height: 2rem;
+  --pg-control-vertical-padding: calc(
+    (2.25rem - var(--p-font-line-height-600) - var(--p-space-050)) / 2
+  );
```

Comment:
Any SASS variables that couldn't be inlined have been converted to CSS Custom Properties with the `--pg-` prefix (and that prefix has been added to our linter along with `--pc-`).